CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ This beta release is built on top of [OpenTelemetry .NET](https://github.com/ope
 - Support configuring `OTEL_*` settings using `App.config` and `Web.config`.
 - Add support for Quartz traces instrumentation.
 - Add support for EntityFrameworkCore traces instrumentations.
+- Add support for custom sdk through `OTEL_DOTNET_AUTO_INJECT_SDK`.
```

Comment:
```suggestion
- Add support for custom OpenTelemetry .NET SDK setup through `OTEL_DOTNET_AUTO_SETUP_SDK`.
```

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2069/commits/7314e8899c9434f53fc74b6a415aa55ad01071cf